include/link.h

```diff
@@ -48,6 +48,28 @@
 #define EXTRACT_LINK_ERRORS(status) \
 (((status) & LINK_STAT_ERRORS) >> LINK_STAT_ERRORS_SHIFT)
 
+#define LINKTYPE_0x1111              0x1111  // trade
```

Comment:
What's the story behind all the unnamed `LINKTYPE_` constants here?  Was it because you weren't able to figure them out?

Reply:
Just documented the ones I did because their usage was obvious in the linkup functions of `cable_club.c`, and the berry blender type is the only link type used by `berry_blender`. I'll get to the rest, but probably outside this PR. I'd rather document the functions they're used in to be sure about their names (and most of those are outside the scope of `cable_club.inc`), so I just commented what they seem to be based on the numbering pattern and where they're used.